.github/workflows/docgen.yml

```
@@ -35,6 +35,10 @@ jobs:
           run: npm i
 
 #Generation
+        - name: Discover ts declarations
+          working-direcory: devextreme
+          run: npm run discover-ts-declarations -- --artifacts ../artifacts --js-scripts ../devextreme/js
+
```

It seems that this lines are redundant:
```suggestion
        - name: Discover ts declarations
          working-direcory: devextreme
          run: npm run discover-ts-declarations -- --artifacts ../artifacts --js-scripts ../devextreme/js

```

next step calls `npm run discover-declarations` script from devextreme, which is: 
```
"discover-declarations": "dx-tools discover --js-scripts ./js --exclude js/renovation/",
```
which calls tsDiscoverer in it's turn:
```
'discover': [runTsDiscoverer, runDiscoverer],
```

---

Yes. Let's rename this script to discover - we have `discover-declarations` in devextreme-internal-tools -> `index.ts` that call Declarations.dll. So such naming is confusing